src/vendor/highlight/demo/theme.twig

```diff
@@ -0,0 +1,5 @@
+<pre{% if shcb %}
+  aria-describedby="shcb-language-demo"
+  data-shcb-language-name="{{languageName}}"
+  data-shcb-language-slug="{{language}}"
+{% endif %}>{% if shcb %}<div>{% endif %}<code class="language-{{name}}">{% if html %}{{html|raw}}{% else %}{{content}}{% endif %}</code>{% if shcb %}</div><small class="shcb-language" id="shcb-language-demo"><span class="shcb-language__label">Code language:</span> <span class="shcb-language__name">{{languageName}}</span> <span class="shcb-language__paren">(</span><span class="shcb-language__slug">{{language}}</span><span class="shcb-language__paren">)</span></small>{% endif %}</pre>
```

Comment:
Non-blocking nit: If this is all on one line because of white space in the `pre` element, we could explore formatting it across multiple lines and using Twig's whitespace trimming functionality: https://symfony.com/blog/better-white-space-control-in-twig-templates

(But I think this is fine for docs)

Reply:
Totally. I'll admit I was tripping over the whitespace trimming a bit and decided to just be lazy about it for the sake of shipping sooner. 😅